packages/rea/rea.0.2.0/opam

```diff
@@ -0,0 +1,23 @@
+opam-version: "2.0"
+synopsis: "Effectful OCaml with Objects and Variants"
+maintainer: "Vesa Karvonen <vesa.a.j.k@gmail.com>"
+authors: "Vesa Karvonen"
+license: "MIT"
+homepage: "https://github.com/polytypic/rea-ml"
+bug-reports: "https://github.com/polytypic/rea-ml/issues"
+dev-repo: "git+https://github.com/polytypic/rea-ml.git"
+depends: [
+  "dune" {>= "3.3.0"}
+  "ocaml" {>= "4.08.0"}
```

Comment:
Hmm... Why is the CI failing with earlier OCaml versions?  With my previous 0.1.0 submission the earlier OCaml versions were skipped by the CI.

Reply:
That was my fault, it’s a change in CI which has been fixed